docs/application/vscode-ext/web.md

```diff
@@ -79,232 +51,237 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
 
-4. Select the **Basic UI** template and click **Next**.
+6. Select the application template, for example **Basic UI**, and click **Next**.
 
     > **Note**
     >
-    > The list is different depending on the type and profile of the previously selected project.
+    > The application template options that appear will differ based on the type and profile of the previously selected project.
 
     ![Select the project](media/tizen_web10.png)
 
-5. To complete the project creation in workspace, enter **Project name** and click **Finish**.
+7. To complete the project creation in workspace, enter **Project name** and click **Finish**.
     > **Note**
     >
-    > packageId is an optional field, it is not mandatory to enter details in this field because the packageId will generate the details automatically.
+    > **Package ID** is an optional field and the details are generated automatically.
 
     ![Enter the name of project](media/tizen_web11.png)
 
-6. If you checked all this, the project is created.
-
-   ![Project is created](media/tizen_web12.png)
-
 ### Create Tizen Certificate
 
 To create Tizen Certificate, follow these steps:
 
-  1. Open **Command Palette** and enter **Create Tizen Certificate**.
-  2. Enter the name of the certificate.
-  3. Enter the password of the certificate.
-  4. Enter the profile name of the certificate. (Extension is .p12)
-  > **Note**
-  > 
-  > If it is created properly, profilename.p12 will be created in tizen-studio/keystore/author.
+  1. In VS Code, click **View** > **Command Palette**.
+  2. In the input field of the **Command Palette** that appears, type **Create Tizen Certificate** and press **Enter**.
+  3. Enter the name of the certificate.
+  4. Enter the password of the certificate.
+  5. Enter the profile name of the certificate. (Extension is .p12)
+      > **Note**
+      > 
+      > profilename.p12 will be created in tizen-studio/keystore/author.
 
 ### Add Tizen Certificate
 
 To add Tizen Certificate, follow these steps:
 
-  1. Open **Command Palette** and enter **Add Tizen Certificate**.
-  2. Enter the name of the certificate.
-  3. Enter the password of the certificate.
-  4. Enter the profile name of the certificate. (Extension is .p12)
-  > **Note**
-  > 
-  > You can verify the added certificate by running **Certificate Manager**.
+  1. In VS Code, click **View** > **Command Palette**.
+  2. In the input field of the **Command Palette** that appears, type **Add Tizen Certificate** and press **Enter**.
+  3. Enter the name of the certificate.
+  4. Enter the password of the certificate.
+  5. Enter the profile name of the certificate. (Extension is .p12)
+      > **Note**
+      > 
+      > To view the added certificate, type **Run Certificate-Manager** in **Command Palette**.
 
 ### Remove Tizen Certificate
 
 To remove Tizen Certificate, follow these steps:
 
 If you want to change the Certificate, you need to remove the added Certificate.
```

Comment:
Line 127: Please rephrase to:

If you want to change the Certificate, you need to remove the added Certificate. To remove Tizen Certificate, follow these steps:

Reply:
done.